src/dams_mcda/ui.R

```diff
@@ -1247,49 +1243,43 @@ ui <- shinyUI(fluidPage(
 		             ),
 
-		             h3('Download Millinocket Results'),
-		             downloadButton("downloadData3", "Download Millinocket")
+		             h3('Download East Millinocket Results'),
+		             downloadButton("downloadData3", "Download East Millinocket")
 				 )
 		),
 
-		tabPanel("Dam 4: East Millinocket Dam",
-		         h2("Results: East Millinocket Dam"),
+		tabPanel("Dam 4: Dolby Dam",
+		         h2("Results: Dolby Dam"),
 		         HTML("Now that you have seen the coordinated multi-dam alternative recommendation, we will drill down to explore the MCDA results for each individual dam.\
                   Remember that these results have been estimated using site-specific data values, so the network-dependent criteria (e.g. sea-run fish habitat area, river recreation)\
 		              values are the average for the possible range, which is actually dependent on the network of dams. You should expect that these results will differ somewhat from the multi-dam\
 		              results.<br>"),
 		         
 		         div(id="dam-4-output",
-		             h3("Table 13. Raw preference scores for East Millinocket Dam"),
+		             h3("Table 13. Raw preference scores for Dolby Dam"),
 		             DT::dataTableOutput("RawPrefsDam4"),
 		             HTML(
 		               "This table of preference data is depicted below."
 		             ),
 
-		             h3("Figure 10. Raw Preference Scores for East Millinocket"),
+		             h3("Figure 10. Raw Preference Scores for Dolby"),
 					       plotOutput("PrefPlot4", height=graph_height, width=graph_width),
 					       HTML(
 					         "<br><b>Results Interpretation</b> for Figure 10: The bars visually represent your preference scores for each decision criterion.\
-		               The scores are pulled directly from your slider bar settings under the East Millinocket Dam tab and are not changed in any way. If you wish to go back and change your settings, please do so before continuing.<br>"
+		               The scores are pulled directly from your slider bar settings under the Dolby Dam tab and are not changed in any way. If you wish to go back and change your settings, please do so before continuing.<br>"
 					       ),
 					       
 					       #raw data table/Matrix
-					       h3("Table 14 Raw data values for East Millinocket Dam"),
+					       h3("Table 14 Raw data values for Dolby Dam"),
 					       DT::dataTableOutput("Dam4RawTable"),
 					       HTML(
 					         "<br><b>Results Interpretation</b> for Table 14. These are the raw data values for the dam development. We include the raw data values here to help make the MCDA calculation more transparent."
 					       ),
 					       # normalized data table/Matrix
-					       h3("Table 15. Normalized data values for East Millinocket Dam"),
+					       h3("Table 15. Normalized data values for Dolb Dam"),
```

Comment:
Dolby

Reply:
Thanks, I'm hoping to get to this momentarily.